cfgov/jinja2/v1/_layouts/base.html

```diff
@@ -122,12 +122,43 @@
     If you come across a script that makes a convincing case to be included in
     the head, then file an issue or PR to discuss including it.
 -->
-    {# Google Tag Manager #}
+
+    <!-- Google Optimize page-hiding snippet -->
+    <style>
```

Comment:
@Scotchester is correct about this being fairly "black box" stuff. I'd imagine it would require considerable work to determine exactly why bugs like this occur. That's not to say it's fine that we have no idea exactly what's going on, but at present I have no suggestions for viable alternatives to letting Google Analytics and Tag Manager pull us around by the nose a bit. :slightly_frowning_face: